assets/src/edit-story/components/form/color/colorBox.js

```diff
@@ -25,8 +25,8 @@ const ColorBox = styled.div.attrs({
 })`
   height: 32px;
   width: 122px;
-  color: ${({ theme }) => rgba(theme.colors.fg.v1, 0.86)};
-  background-color: ${({ theme }) => rgba(theme.colors.bg.v0, 0.3)};
+  color: ${({ theme }) => rgba(theme.colors.fg.v1, 0.86)} !important;
+  background-color: ${({ theme }) => rgba(theme.colors.bg.v0, 0.3)} !important;
```

Comment:
These seemed to be overwritten by default browser values for input.

Reply:
Or WordPress styles perhaps? See #349

Reply:
Although this is just a `div` 🤔

Reply:
Ah, but in `opacityPreview.js` it is used as an `input`.

Reply:
Is it WP or browser stylesheet. Seems very unusual. Especially this is a `div` used here? If we are conflicting with some `forms.css` - can we resolve this generically? It might be worth moving into a separate bug/pull request, since otherwise seems unrelated to text stuff?

Reply:
Yes, it's the WP `forms.css` that it's conflicting with. One option could perhaps be just disabling the `forms.css` for our editor, not sure if it's needed for anything else that we couldn't add ourselves.

This has a separate issue, added a comment there, too: https://github.com/google/web-stories-wp/issues/349